app/views/docs/databases-relationships.phtml

```diff
@@ -0,0 +1,880 @@
+        <tr>
+            <th style="width: 250px">Type</th>
+            <th>Description</th>
+        </tr>
+    </thead>
+    <tbody>
+        <tr>
+            <td>One-way</td>
+            <td>The relationship is only visible to one side of the relation. This is similar to a tree data structure, where a tree has a reference to all of its leaves, but each leaf does not have a reference to its tree root.</td>
+        </tr>
+        <tr>
+            <td>Two-way</td>
+            <td>The relationship is visible to both sides of the relationship. This is similar to a graph data structure, where each node has references to all its edges, and all its edges can reference the nodes it connects.</td>
+        </tr>
+    </tbody>
+</table>
+
+<h3><a href="/docs/databases-relations#types" id="types">Types</a></h3>
+<p>Appwrite provides four different relationship types to enforce different associative rules between documents.</p>
+<table cellspacing="0" cellpadding="0" border="0" class="full margin-bottom-large">
+    <thead>
+        <tr>
+            <th style="width: 250px">Type</th>
+            <th>Description</th>
+        </tr>
+    </thead>
+    <tbody>
+        <tr>
+            <td>One-to-one</td>
+            <td>A document can only be related to one and <b>only</b> one document. If you try to relate multiple documents in a one-to-one relationship, Appwrite throws an error. For example, <b>one</b> user has <b>one</b> profile.</td>
+        </tr>
+        <tr>
+            <td>One-to-many</td>
+            <td>A document can be related to many other documents. For example, <b>one</b> user can create <b>many</b> posts.</td>
+        </tr>
+        <tr>
+            <td>Many-to-one</td>
+            <td>Many documents can be related to a single document. For example, <b>many</b> posts can share <b>one</b> author.</td>
+        </tr>
+        <tr>
+            <td>Many-to-many</td>
+            <td>A document can be related to many other documents. For example, a user can have <b>many</b> friends, and <b>many</b> users can share the same friend.</td>
+        </tr>
+    </tbody>
+</table>
+
+<h3><a href="/docs/databases-relations#on-delete" id="on-delete">On-Delete Behavior</a></h3>
+<p>
+    Appwrite also allows you to define the behavior of a relationship when a document is deleted.
+</p>
+<table cellspacing="0" cellpadding="0" border="0" class="full margin-bottom-large">
+    <thead>
+        <tr>
+            <th style="width: 250px">Type</th>
+            <th>Description</th>
+        </tr>
+    </thead>
+    <tbody>
+        <tr>
+            <td><code>Restrict</code></td>
+            <td>If a document has at least one related document, it cannot be deleted.</td>
+        </tr>
+        <tr>
+            <td><code>Cascade</code></td>
+            <td>If a document has related documents, when it is deleted, the related documents are <b>also</b> deleted.</td>
+        </tr>
+        <tr>
+            <td><code>Set null</code></td>
```

Comment:
These aren't the actual code values so we probably don't want to wrap with `<code>` tags.